src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -181,19 +181,19 @@ INSTANTIATE_TEST_SUITE_P(smoke_OVClassSetDevicePriorityConfigPropsTest,
                                             ::testing::ValuesIn(multiConfigs)));
 
 const std::vector<ov::AnyMap> configsDeviceProperties = {
-    {ov::device::properties("CPU", ov::num_streams(3))},
-    {ov::device::properties(ov::AnyMap{{"CPU", ov::AnyMap{ov::num_streams(3)}}})}};
+    {ov::device::properties("CPU", ov::num_streams(2))},
```

Comment:
How about 1 processor situation?

Reply:
In current design, CPU inference is always use 1 stream on 1 processor platform. 

This test case is not valid on 1 processor situation.